### PR TITLE
Updating stdout for output files written

### LINKED ISSUE #237: Print out names of all written files
# Description of feature

Currently, when writing volume and surface solutions the file names are printed to screen, but the same is not done for other files.

### PR DESCRIPTION
**Pull request by eirikurj**
## Purpose
As described in https://github.com/mdolab/adflow/issues/237, filenames for different output files are not consistently written out. This PR addresses these issues. Additionally, time-accurate and time-spectral files should now also be written consistently to stdout.

Closes #237 

Expected output for steady cases
```
#
# Writing grid file(s):
#    outdir/wing_000_vol.cgns
# Grid file(s) written
#
#
# Writing volume solution file(s):
#    outdir/wing_000_vol.cgns
# Volume solution file(s) written
#
#
# Writing surface solution file(s):
#    outdir/wing_000_surf.cgns
# Surface solution file(s) written
#
#
# Writing slices file(s): 
#    outdir/wing_000_slices.dat
# Slices file(s) written
#
#
# Writing lift distribution file(s):
#    outdir/wing_000_lift.dat
# Lift distribution file(s) written
#
#
# Writing tecplot surface file(s):
#    outdir/wing_000_surf.plt
# Tecplot surface file(s) written
```

Expected output for one time-step
```
# Writing grid file(s):
#    outdir/wing_000_vol.cgnsTimestep2
#    outdir/wing_000_vol.cgnsTimestep1
# Grid file(s) written
#
#
# Writing volume solution file(s):
#    outdir/wing_000_vol.cgnsTimestep2
#    outdir/wing_000_vol.cgnsTimestep1
# Volume solution file(s) written
#
#
# Writing surface solution file(s):
#    outdir/wing_000_surf.cgnsTimestep0002
# Surface solution file(s) written
#
#
# Writing slices file(s): 
#    outdir/wing_000_slices_Timestep0002.dat
# Slices file(s) written
#
#
# Writing lift distribution file(s):
#    outdir/wing_000_lift_Timestep0002.dat
# Lift distribution file(s) written
#
#
# Writing tecplot surface file(s):
#    outdir/wing_000_surf_Timestep0002.plt
# Tecplot surface file(s) written
#

```

## Expected time until merged
No rush

## Type of change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
Please test different problems and combinations to make sure it works as expected.

## Checklist

- [ ] I have run `flake8` and `black` to make sure the Python code adheres to PEP-8 and is consistently formatted
- [ ] I have formatted the Fortran code with `fprettify` or C/C++ code with `clang-format` as applicable
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: src/output/tecplotIO.F90 [hunk 1]
```diff
@@ -312,12 +312,12 @@ subroutine writeSlicesFile(fileName, nodalValues)
...
           print "(a)", "#"
-          print "(a)", "# Writing slices file(s) ..."
+          print "(a)", "# Writing slices file(s): "
        endif
 
-       do sps=1,nTimeIntervalsSpectral
+       do sps=1, nTimeIntervalsSpectral
 
-          ! If it is time spectral we need to agument the filename
+          ! If it is time spectral we need to augument the filename
```
Comment (marcomangano) [inline src/output/tecplotIO.F90:320]: "augment"

Comment (eirikurj) [reply src/output/tecplotIO.F90]: fixed

**Comment by eirikurj** [comment]
Converting to draft for now. Will provide more detail later.

**Comment by eirikurj** [comment]
Just to clarify, this one is ready for review

**Review by marcomangano — DISMISSED** [review]

**Review by marcomangano — APPROVED** [review]

**Review by ArshSaja — APPROVED** [review]
The printing looks clean!

**Pull request merged**